pyrometheus/__init__.py

```diff
@@ -353,10 +352,15 @@ def _pyro_make_array(res_list):
     \"""
 
     from numbers import Number
-    all_numbers = all(isinstance(e, Number) for e in res_list)
+    all_numbers = all(
+        isinstance(e, Number)
+        or (isinstance(e, usr_np.ndarray) and e.shape == ())
+        for e in res_list)
```

Comment:
For my education: What does Jax do without this special treatment?

Reply:
The entries in `res_list` are something like `jaxlib.DeviceArray(number, dtype=float32)` and if they're not recognized as scalars, they're put inside an object array in the next lines.

I don't recall if that messed up anything important or just at the end an `np.allclose` failed. I'm guessing it should have a similar effect as removing the special handling for numbers (?).